src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <title>Product cards</title>
+  </head>
+  <body>
+    <div class="conteiner">
+      <div class="card" data-qa="card">
+        <img
+          src="/images/imac.jpeg"
+          alt="imac"
+          class="card__immage"
+        >
+
+        <h2 class="card__product-name">
+           Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">
+          Код товара: 195434
+        </p>
+
+        <div class="card__feedback">
+          <div class="stars stars--4">
+            <button class="stars__button"></button>
+            <button class="stars__button"></button>
+            <button class="stars__button"></button>
+            <button class="stars__button"></button>
+            <button class="stars__button"></button>
+          </div>
+
+          <div class="card__feedback-rate">
+            Отзывов: 5
+          </div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-name">Цена:</div>
+          <div class="card__price-number">69 999 грн</div>
+        </div>
+
+        <div class="card__buy">
+          <a
+          href="/"
+          class="card__buy-button"
+          data-qa="hover"
+          >
+          Купить
+          </a>
+        </div>
```

Comment:
Wrapper redundant here, also fix code style in all cases

```suggestion
          <a
            href="/"
            class="card__buy-button"
            data-qa="hover"
          >
            Купить
          </a>
```